pubspec.yaml

```diff
@@ -5,70 +5,21 @@ homepage: https://github.com/linagora/jmap-dart-client.git
 
 environment:
   sdk: ">=2.19.2 <3.0.0"
-  flutter: ">=3.0.0"
 
 dependencies:
-  flutter:
-    sdk: flutter
-
-  equatable: 2.0.5
-
+  build_runner: 2.3.3
+  built_collection: 5.1.1
   dartz: 0.10.1
-
-  quiver: 3.2.1
-
   dio: 5.0.0
-
+  equatable: 2.0.5
+  http_parser: 4.0.2
   json_annotation: 4.8.0
-
-  build_runner: 2.3.3
-
   json_serializable: 6.6.1
-
-  built_collection: 5.1.1
-
-  http_parser: 4.0.2
```

Comment:
Please do not format its order.

Reply:
Resolved in commit 5cf5328be2336319892d273916766a036a5e08f5